intel-retail/automated-vending
# fix: Dependabot reference updates

## PR Checklist



## What are you changing?

Update go.mod based on dependabot recommendations.

## Issue this PR will close

close: #173 
close: #172 
close: #171 
close: #170 
close: #169 
close: #168 
close: #167 
close: #166 
close: #164 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- update branch
- make docker
- make run
- ./smoke-test.sh
- See no errors (all 200 responses)

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-vending )
